### DISCUSSION TITLE
yuque渲染出出错

### DISCUSSION THREAD

**Discussion by lymyxhw** [F9y4ng/GreasyFork-Scripts#229] [问答专区(Question / Answer)]
https://www.yuque.com/chentaotao-cf9fr/gthfy4/wrc8ef
![image](https://github.com/F9y4ng/GreasyFork-Scripts/assets/26971401/d310fb50-6bed-45dd-ae14-7db8fd3cab42)

**Comment by F9y4ng** [comment]
#42
